pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -198,15 +199,15 @@ func TestValidateKubeSchedulerConfigurationV1beta2(t *testing.T) {
 
 	extenderDuplicateManagedResource := validConfig.DeepCopy()
 	extenderDuplicateManagedResource.Extenders[0].ManagedResources = []config.ExtenderManagedResource{
-		{Name: "foo", IgnoredByScheduler: false},
-		{Name: "foo", IgnoredByScheduler: false},
+		{Name: "example.com/foo", IgnoredByScheduler: false},
```

Comment:
why did this have to change?
Did it have an extra validation error?

Reply:
foo or bar is an invalid ExtenderResource name. The original version returned additional invalid resource errors.